wls-auth-service/src/main/java/de/muenchen/oss/wahllokalsystem/authservice/configuration/SecurityConfiguration.java

```diff
@@ -31,15 +53,36 @@
 public class SecurityConfiguration {
 
     @Autowired
-    private RestTemplateBuilder restTemplateBuilder;
+    private CustomUsernamePasswordAuthenticationFilter customUsernamePasswordAuthenticationFilter;
 
     @Autowired
-    private CustomUsernamePasswordAuthenticationFilter customUsernamePasswordAuthenticationFilter;
+    private UserService userService;
 
-    @Value("${security.oauth2.resource.user-info-uri}")
-    private String userInfoUri;
+    @Value("${service.config.oauth2.jwk.rsa.init.seed}")
```

Comment:
Wo kommt dieser Wert in Produktion her?

Reply:
Ok, gefunden: "It is planned to use external files for the key. See issue https://github.com/it-at-m/Wahllokalsystem/issues/605"